README.md

```diff
@@ -16,35 +16,16 @@
     - [Getting a list of schools: `GET https://servers.somtoday.nl/organisaties.json`](#getting-a-list-of-schools-get-httpsserverssomtodaynlorganisatiesjson)
   - [Authentication / authorization](#authentication--authorization)
     - [Fetching the access token: `POST /oauth2/token`](#fetching-the-access-token-post-oauth2token)
-      - [Parameters](#parameters)
-      - [Returns](#returns)
-      - [Example](#example)
     - [Refreshing the token: `POST /oauth2/token`](#refreshing-the-token-post-oauth2token)
-      - [Parameters](#parameters-1)
-      - [Returns](#returns-1)
-      - [Example](#example-1)
   - [Fetching information](#fetching-information)
     - [Current student(s): `GET /rest/v1/leerlingen`](#current-students-get-restv1leerlingen)
-      - [Parameters](#parameters-2)
-      - [Returns](#returns-2)
-      - [Example](#example-2)
     - [Student by ID: `GET /rest/v1/leerlingen/[id]`](#student-by-id-get-restv1leerlingenid)
-      - [Parameters](#parameters-3)
-      - [Returns](#returns-3)
-      - [Example](#example-3)
     - [Grades: `GET /rest/v1/resultaten/huidigVoorLeerling/[id]`](#grades-get-restv1resultatenhuidigvoorleerlingid)
-      - [Parameters](#parameters-4)
-      - [Returns](#returns-4)
     - [Schedule: `GET /rest/v1/afspraken`](#schedule-get-restv1afspraken)
-      - [Parameters](#parameters-5)
-      - [Returns](#returns-5)
-      - [Example](#example-4)
     - [Absence Reports: `GET /rest/v1/absentiemeldingen`](#absence-reports-get-restv1absentiemeldingen)
-      - [Parameters](#parameters-6)
-      - [Returns](#returns-6)
     - [Study Guides: `GET /rest/v1/studiewijzers`](#study-guides-get-restv1studiewijzers)
-      - [Parameters](#parameters-7)
-      - [Returns](#returns-7)
+    - [Study Guides: `GET /rest/v1/account`](#account-get-restv1account--get-restv1accountid)
+    - [Study Guides: `GET /rest/v1/schooljaren`](#account-get-restv1schooljarenhuidig)
```

Comment:
This link no longer works

Reply:
Wait really, like since today?

Reply:
The header has been changed, but the link hasn't

Reply:
well i tested the api endpoint and it seems to work for me, it doesn't have any headers
![Schermafbeelding 2022-10-18 194430](https://user-images.githubusercontent.com/78482881/196505689-44dec685-3a79-456c-b106-c8980281cf74.jpg)

Reply:
I meant the header in the documentation :)

Reply:
Ohhh, my bad. But I don't know how to change it in this pull request... first time.